src/main/java/nextstep/optional/User.java

```diff
@@ -33,7 +35,11 @@ public static boolean ageIsInRange1(User user) {
     }
 
     public static boolean ageIsInRange2(User user) {
-        return false;
+        return Optional.ofNullable(user)
+                .filter(item -> Optional.ofNullable(item.getAge())
+                        .filter(age -> age >= 30 && age <= 45)
+                        .isPresent())
+                .isPresent();
```

Comment:
```suggestion
                .map(User::getAge)
                .filter(age -> age >= 30)
                .filter(age -> age <= 45)
                .isPresent();
```

Method Reference를 적극적으로 활용해보는 것은 어떨까요

Reply:
👍  좋아보이네요. 감사합니다.